x-pack/plugins/ml/public/embeddables/single_metric_viewer/single_metric_viewer_embeddable_factory.tsx

```diff
@@ -88,6 +88,28 @@ export const getSingleMetricViewerEmbeddableFactory = (
 
       const api = buildApi(
         {
+          isEditingEnabled: () => true,
+          getTypeDisplayName: () =>
+            i18n.translate('xpack.ml.singleMetricViewerEmbeddable.typeDisplayName', {
+              defaultMessage: 'Single metric viewer',
+            }),
+          onEdit: async () => {
```

Comment:
When editing an existing embeddable, can all the form controls be initialized to have their currently saved values, rather than showing (empty) defaults:

<img width="683" alt="Screenshot 2024-05-07 at 17 52 19" src="https://github.com/elastic/kibana/assets/7405507/2fa78113-c7f9-4899-ad6c-df45ec704b1b">

Reply:
Good catch - fixed in eef22339907c62be64fb49d358bfeb350cdfd72a